Tests/MusicNotationCoreTests/TupletTests.swift

```diff
@@ -352,6 +352,16 @@ class TupletTests: XCTestCase {
             )
         }
     }
+    
+    func testInitSuccessForComplexCompound() {
```

Comment:
You wrote this brand new test?

If so, it should probably fail somehow and be set to the same KNOWNISSUE prefix. 

If the only way to get it to fail is testing the print, then you should be able to compare the debug print statement. However, if the print fails, I’m assuming maybe the structure of it may be wrong too. It’d be preferable to make it fail without having to compare strings.

Reply:
Will be in another PR